package.json

```diff
@@ -39,6 +39,7 @@
     "cross-fetch": "^3.1.5",
     "dotenv": "^16.0.2",
     "ethers": "^5.7.1",
+    "ethers-v6": "npm:ethers@latest",
```

Comment:
Is ethers-v6 strictly required? Would like to keep dependencies clean but we can use it if you need to